nautobot/core/forms/forms.py

```diff
@@ -121,7 +121,7 @@ class BulkRenameForm(forms.Form):
     """
 
     find = forms.CharField()
-    replace = forms.CharField()
+    replace = forms.CharField(required=False, strip=False)
```

Comment:
This allows you to have a blank replacement string, or to have leading and trailing whitespace on your replacement string. Examples:

`Interface1`

Replace `^Interface` with blank string to change the name to just `1`
Replace `^Interface` with `Interface ` to change the name to `Interface 1`